Sources/Models/Conversion.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Conversion.swift
+//  Rover
+//
+//  Created by Chris Recalis on 2020-06-02.
+//  Copyright © 2020 Rover Labs Inc. All rights reserved.
+//
+
+
+public struct Duration: Decodable {
+    public enum Unit: String, Decodable {
+        case seconds = "s"
+        case minutes = "m"
+        case hours = "h"
+        case days = "d"
+    }
+    
+    public var value: Int
+    public var unit: Unit
+    
+    
+    public init(value: Int, unit: Unit) {
+        self.value = value
+        self.unit = unit
+    }
+    
+    public var timeInterval: TimeInterval {
+        let base: Int
+        switch self.unit {
+        case .seconds:
+            base = 1
+        case .minutes:
+            base = 60
+        case .hours:
+            base =  60 * 60
+        case .days:
+            base = 24 * 60 * 60
+        }
+        return TimeInterval(self.value * base)
+    }
```

Comment:
Should this belong in campaigns as an extension rather on the model itself?

Reply:
If extension is possible

Reply:
@chrisrecalis if you're only using it in rover-campaigns-ios, then yeah, you can totally do it with an extension there instead. Having it here means anything using a Conversion value from rover-ios can calculate the TimeInterval value. It's up to you!

Reply:
@orospakr what do you think is the best solution? I attempted to add an extension in `rover-campaigns-ios` but wasn't able to